backend/StreamServerApp/subtitles.py

```diff
@@ -16,21 +16,49 @@ def init_cache():
             'filename': 'cachefile.dbm'}, replace_existing_backend=True)
 
 
-def handle_subliminal_download(video, video_path, langage):
-    best_subtitles = download_best_subtitles([video], {Language(langage)})
+def handle_subliminal_download(video, video_path):
+    """ # Download the best subtitles in french and english
+        Args:
+        video : Name of video
+        video_path: absolute path to videos
+    """
+    best_subtitles = download_best_subtitles([video], {Language('fra'),Language('eng')})
     if best_subtitles[video]:
-        best_subtitle = best_subtitles[video][0]
-        value = save_subtitles(video, [best_subtitle], encoding='utf8')
-        if len(value) > 0:
+
+        if best_subtitles[video][0].language == Language('eng') :
```

Comment:
Have you checked that you need to do this check? Don't the subtitles appear in the same order you asked them?
In any case, I think it would be better for readability and modularity to iterate through all retrieved subtitles and make the decision regarding their languages after. E.g.:
```python
for retrieved_subtitle in best_subtitles[video]:
    value = save_subtitles(video, [retrieved_subtitle ], encoding='utf8')   # here 'value' should be more explicit, I don't know to what it corresponds to. I know it was like that in the original code, but the variable naming should be improve.
```